libqtile/backend/wayland/cffi/build.py

```diff
@@ -30,5 +31,13 @@
 ffi.cdef(libinput.CDEF)
 ffi.cdef(cairo_buffer.CDEF)
 
+
+def ffi_compile(verbose: bool = False) -> None:
+    # Always output compiled file to correct folder, regardless from where the script is called
+    ffi.compile(
+        tmpdir=Path(__file__).parent.parent.parent.parent.parent.as_posix(), verbose=verbose
```

Comment:
I have to admit, I don't understand why this works, even after reading https://cffi.readthedocs.io/en/latest/cdef.html#ffibuilder-compile-etc-compiling-out-of-line-modules

Can you put a comment here about what directory which files are supposed to land in?

(It is strange to me that `tmpdir` changes the directory of the final output, but it does seem to...)